### PR TITLE
merge Silva references

### PR DESCRIPTION
**Pull request by aortiz24**
I have incorporated recommendations from #6 and #15

### CODE REVIEW

File: mothur_otu.bat [hunk 1]
```diff
@@ -21,7 +21,7 @@ summary.seqs(fasta=analysis.trim.unique.fasta, name=analysis.trim.names)
 
 # align our data to silva reference
 # output: analysis.trim.unique.align, analysis.trim.unique.align.report, analysis.trim.unique.flip.accnos 
-align.seqs(fasta=analysis.trim.unique.fasta, reference=silva/silva.bacteria.pcr.fasta, flip=T, processors=6)
+align.seqs(fasta=analysis.trim.unique.fasta, reference=Silva.nr_v128/silva.nr_v128.align, flip=T, processors=6)
```
Comment (k8hertweck) [inline mothur_otu.bat]: This should be the `silva.nr_128.pcr.align` file

**Comment by k8hertweck** [comment]
I just realized I hadn't published the review for the previous commits. Take a look at them and let me know if you've accommodate them.

Also, the last commits you pushed add two files that shouldn't be there: `mothur_SilvaRef.bat~` and `mothur_otu.bat~`. Please reset to before these commits.

File: mothur_SilvaRef.bat [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
 # rename downloaded database
-#system(cp Silva.seed_128 SilvaSeed.fasta)
-#or using the silva reference files from the MiSeq SOP: silva.bacteria.fasta 
+#system(cp Silva.seed_128.align SilvaSeed.fasta)
+#or using the updated silva reference files from the MiSeq SOP: silva.nr_v128.align 
 
 # aligns known sequence to complete database
-# output: E.coliV1V3.align, E.coliV1V3.align.report, E.coliV1V3.flip.accnos file
-align.seqs(fasta=silva/E.coliV1V3.fas, template=silva/silva.bacteria.fasta)
+# move silva.nr_128.align to the silva directory
```
Comment (k8hertweck) [inline mothur_SilvaRef.bat]: This information should be included in the README to describe how the project directory is set up

**Comment by aortiz24** [comment]
I have completed the additional edits to the PR.

**Pull request merged**